src/core/toxoptions.cpp

```diff
@@ -3,6 +3,10 @@
  * Copyright © 2024-2025 The TokTok team.
  */
 
+#ifdef TOX_HIDE_DEPRECATED
+#undef TOX_HIDE_DEPRECATED
+#endif
```

Comment:
isn't this supposed to be opt-in and not opt-out?

Reply:
Removed.